engine/src/main/scala/cromwell/engine/workflow/WorkflowDockerLookupActor.scala

```diff
@@ -104,10 +105,18 @@ class WorkflowDockerLookupActor private[workflow](workflowId: WorkflowId,
 
   whenUnhandled {
     case Event(DockerHashActorTimeout(request), data) =>
-      val reason = new Exception(s"Timeout looking up docker hash")
-      data.hashRequests(request) foreach { _ ! WorkflowDockerLookupFailure(reason, request) }
-      val updatedData = data.copy(hashRequests = data.hashRequests - request)
-      stay() using updatedData
+      val reason = new Exception(s"Timeout looking up hash for Docker image ${request.dockerImageID}")
```

Comment:
Can we also add `"... in state ${$stateName}"`?

Reply:
a syntactically correct version of that, sure 😉